src/mqt/predictor/ml/helper.py

```diff
@@ -0,0 +1,211 @@
+import sys
+
+if sys.version_info < (3, 10, 0):
+    import importlib_resources as resources
+else:
+    from importlib import resources
+
+from pathlib import Path
+
+import numpy as np
+from joblib import dump
+from qiskit import QuantumCircuit
+
+from mqt.predictor import utils
+
+
+def get_path_training_data_ML():
+    return resources.files("mqt.predictor") / "ml" / "training_data"
+
+
+def get_path_trained_model_ML():
+    return get_path_training_data_ML() / "trained_model"
+
+
+def get_path_training_circuits_ML():
+    return get_path_training_data_ML() / "training_circuits"
+
+
+def get_path_training_circuits_ML_compiled():
+    return get_path_training_data_ML() / "training_circuits_compiled"
```

Comment:
As mentioned elsewhere, i dislike the `_ML` suffix. The `ml` module alone suffices to specify which part of the project a function belongs to.

Reply:
done.